README.md

```diff
@@ -53,7 +53,7 @@ Alternatively, skip all the steps by using [![Gitpod Ready-to-Code](https://img.
 
 Create a file named `your-username.json` in the directory `public/data` with the following content:
 
-Optional fields: `links` and `milestones`
+Optional fields: `links`,`milestones` and `type(personal | community)`
```

Comment:
```suggestion
Optional fields: `links`, `milestones` and, `type(personal | community)`
```

Reply:
This is incorrect, the Oxford comma follows the second to last item in the list and precedes the `and`.

```suggestion
Optional fields: `links`, `milestones`, and `type(personal | community)`
```

Reply:
So Grammarly is incorrect? 🤔

Reply:
Must be a bug. Even [their article](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) shows it as I've indicated.

Reply:
Oxford comma precedes the 'and' but is also entirely optional in English. Sometimes it helps the reader understand but in this case it's really not necessary.

Reply:
```suggestion
Optional fields: `links`, `milestones` and `type(personal | community)`
```

Reply:
I think relying on Grammarly isn't a good idea(except for spelling checks)

Reply:
We still need the space.